src/database/migrations/20210104113400-course-audit.ts

```diff
@@ -0,0 +1,36 @@
+import { DataTypes } from 'sequelize';
```

Comment:
Why do we have these as two different imports?
Skeleton will help with this in the future